### PR TITLE
Injured Reserve model, migrations, and index

### LINKED ISSUE #204: List all injured reserve transaction for each league
Once admins can create IR transactions, those should be displayed to the owners.

### PR DESCRIPTION
**Pull request by axelclark**
* Creates the migration and model for Injured Reserve
* Adds functionality for admin to update an Injured Reserve transaction
* Lists injured reserve transactions for owners to see
* Closes #204 
* See #174

### CODE REVIEW

File: web/models/injured_reserve.ex [hunk 1]
```diff
@@ -0,0 +1,47 @@
+defmodule Ex338.InjuredReserve do
```
Comment (credoci) [inline web/models/injured_reserve.ex]: Modules should have a @moduledoc tag.

File: web/admin/injured_reserve.ex [hunk 2]
```diff
@@ -0,0 +1,17 @@
+defmodule Ex338.ExAdmin.InjuredReserve do
```
Comment (credoci) [inline web/admin/injured_reserve.ex]: Modules should have a @moduledoc tag.

File: web/models/injured_reserve.ex (hunk 1 above)
Comment (axelclark) [inline web/models/injured_reserve.ex]: fixed

File: web/admin/injured_reserve.ex (hunk 2 above)
Comment (axelclark) [inline web/admin/injured_reserve.ex]: fixed

**Pull request merged**